pkg/cmd/pulumi/destroy.go

```diff
@@ -134,20 +149,11 @@ func newDestroyCmd() *cobra.Command {
 				return result.FromError(fmt.Errorf("gathering environment metadata: %w", err))
 			}
 
-			sm, err := getStackSecretsManager(s)
-			if err != nil {
-				return result.FromError(fmt.Errorf("getting secrets manager: %w", err))
-			}
-
```

Comment:
We now get the secretsManager directly from the snapshot that the backend recorded rather than the current project.

unsure if there are edge cases where we want users to be able to have the local directory dictate destroy behavior rather than using the backend's state to unwind.